service-registry/src/main/java/org/apache/servicecomb/serviceregistry/version/VersionRuleUtils.java

```diff
@@ -54,7 +54,6 @@ public static VersionRule create(String strVersionRule) {
         return versionRule;
       }
     }
-
-    throw new IllegalStateException("never run to here");
+    throw new IllegalStateException("config service_description.version is invalid ");
```

Comment:
why versionRule relate to "service_description.version"?

Reply:
I mistake version and versionRule at first. And forget to delete it